data/implementations/index.json

```diff
@@ -481,25 +1166,169 @@
+                        "x": "40TexHWb6XTyuShaqhiazvmfxyK5zibbtOBXsQFKJg8",
+                        "y": "SGIGDSRHOOYJntO1lIapw_vR1FP7SPBlmA_2aM9HoFU"
+                      }
+                    },
+                    {
+                      "id": "did:example:123#key-3",
+                      "type": "JsonWebKey2020",
+                      "controller": "did:example:123",
+                      "publicKeyJwk": {
+                        "kty": "EC",
+                        "crv": "P-384",
+                        "x": "BiU_mGfa3uWMKrC4Q6EFvM5D5Qiz2orm7ABlIaC1iJWOuapQC9U_fbqrKwRFRepf",
+                        "y": "_23qoYv94V-PSWzqQMnUqq1nu_MdE4fEIaAhCCYplAwlp4c3LKZDtkgQaPdF4kIT"
+                      }
+                    },
+                    {
+                      "id": "did:example:123#key-4",
+                      "type": "JsonWebKey2020",
+                      "controller": "did:example:123",
+                      "publicKeyJwk": {
+                        "kty": "RSA",
+                        "n": "oF_3bdq-pORqmzx2zROkrhVzAPQQ9JmDh1tjKvKgFh-1M5gywClzn0LoWkIiqqIiLOWXkfGRsqvi4VLbHQ_9WZza54EgIo_cyrZnXaIJCEFGQfwDK5-HqDqrgPCw3DFths04LY59LJ2aNxwgbu5otkQ2Vp8bU-U3_Hr5qz_U9xZjOev3OyD7kfBELAFXYHqnSfSJBJ-YxFjo-Plicjfha14U6C-hMeJlDu-Nwj-hCEHdDGK01_go9vIzA745xwAFfeyUZVhTpu6WLSn4RN9vRroLRjO5gvmHUZUquGh-sR4KNLebOeWcyxS5QEAbsYR82YsEWucLTdpFSdu1RVu8nw",
+                        "e": "AQAB"
+                      }
+                    }
+                  ],
+                  "assertionMethod": [
+                    "did:example:123#key-0",
+                    "did:example:123#key-1",
+                    "did:example:123#key-2",
+                    "did:example:123#key-3",
+                    "did:example:123#key-4"
+                  ],
+                  "authentication": [
+                    "did:example:123#key-0",
+                    "did:example:123#key-1",
+                    "did:example:123#key-2",
+                    "did:example:123#key-3",
+                    "did:example:123#key-4"
+                  ]
+                }
+              }
+            }
+          ]
+        }
       }
     }
   },
+  "spruce": {},
   "transmute": {
-    "credential-0--key-0-ed25519.json": {
-      "vector": {
+    "credential-0--key-0-ed25519.vc-jwt.json": {
+      "vector": "vc-jwt",
+      "vectorContent": "eyJhbGciOiJFZERTQSIsImtpZCI6ImRpZDpleGFtcGxlOjEyMyNrZXktMCJ9.eyJpc3MiOiJkaWQ6ZXhhbXBsZToxMjMiLCJzdWIiOnt9LCJ2YyI6eyJAY29udGV4dCI6WyJodHRwczovL3d3dy53My5vcmcvMjAxOC9jcmVkZW50aWFscy92MSIsImh0dHBzOi8vdzNpZC5vcmcvc2VjdXJpdHkvc3VpdGVzL2p3cy0yMDIwL3YxIl0sInR5cGUiOlsiVmVyaWZpYWJsZUNyZWRlbnRpYWwiXSwiaXNzdWVyIjoiZGlkOmV4YW1wbGU6MTIzIiwiaXNzdWFuY2VEYXRlIjoiMjAyMS0wMS0wMVQxOToyMzoyNFoiLCJjcmVkZW50aWFsU3ViamVjdCI6e319LCJuYmYiOjE2MDk1MjkwMDR9.ZXlKaGJHY2lPaUpGWkVSVFFTSXNJbUkyTkNJNlptRnNjMlVzSW1OeWFYUWlPbHNpWWpZMElsMTkuLnhEZFdQcWFpOVFKQTlZY1N2WVU3aks0LVJ0WUpOOXdWTHdmYV9CLTFIX0I1S1licDJaN0pNdWFsVGR5SG0xazB3UWxVa3ZrbXM3bzFmUk92X1RMWkRn",
```

Comment:
The signature part of this JWT does not appear to be a signature corresponding to the header and payload of the JWT:
```
XlKaGJHY2lPaUpGWkVSVFFTSXNJbUkyTkNJNlptRnNjMlVzSW1OeWFYUWlPbHNpWWpZMElsMTkuLnhEZFdQcWFpOVFKQTlZY1N2WVU3aks0LVJ0WUpOOXdWTHdmYV9CLTFIX0I1S1licDJaN0pNdWFsVGR5SG0xazB3UWxVa3ZrbXM3bzFmUk92X1RMWkRn
```
Instead of decoding to a 64-byte EdDSA signature, it decodes to the following, which is itself a JWT with detached unencoded payload:
```
eyJhbGciOiJFZERTQSIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..xDdWPqai9QJA9YcSvYU7jK4-RtYJN9wVLwfa_B-1H_B5KYbp2Z7JMualTdyHm1k0wQlUkvkms7o1fROv_TLZDg
```

Reply:
Thanks, pretty sure I broke this recently, trying to address secp256k1 lower s related issues... I opened this issue here to track the fix: https://github.com/transmute-industries/verifiable-data/issues/116